dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json

```diff
@@ -968,7 +968,7 @@
     {
      "type": "html",
      "name": "thank_you_header",
-     "html": "<img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"https://www.jotform.com/uploads/bakerfranke/form_files/codeorg_heart_you.5cc89a7c7f2f20.88587690.png\" />\n<p>Thanks from Code.org for your time, commitment and passion for teaching {workshop_course} this year.</p>\n<p><strong>You are making a difference for your students.</strong> We're looking forward to what's next. Onward!</p>"
+     "html": "<img style=\"display: block; margin-left: auto; margin-right: auto;\" src=\"<img src='/shared/images/foorm/codeheartyou.png'/>\" />\n<p>Thanks from Code.org for your time, commitment and passion for teaching {workshop_course} this year.</p>\n<p><strong>You are making a difference for your students.</strong> We're looking forward to what's next. Onward!</p>"
```

Comment:
Slight issue with the syntax here, this will fix it (put single quotes instead of escaping double quotes and remove extra img tag):
```suggestion
     "html": "<img style='display: block; margin-left: auto; margin-right: auto;', src='/shared/images/foorm/codeheartyou.png'/>\n<p>Thanks from Code.org for your time, commitment and passion for teaching {workshop_course} this year.</p>\n<p><strong>You are making a difference for your students.</strong> We're looking forward to what's next. Onward!</p>"
```